ansible/deploy.yml

```diff
@@ -33,3 +33,17 @@
       include: render_and_copy_to_s3.yml file={{item}} permission="public-read"
       with_items:
         - configuration/configuration.json
+
+    - name: create EFS volume for persistent home dirs   
+      efs:
+        state: present
+        name: ATMO-prod 
+        tags:
+            app: telemetry-analysis
+        targets:
+            - subnet_id: subnet-63f72706
+              security_groups: [ "sg-0964dc71" ]
```

Comment:
How are these security groups created today? I generally shy away from hardcoding values. If this security group is created via Cloudformation or something else maybe we can call AWS API to figure out which one to use.